src/main/java/org/reactome/orthoinference/Main.java

```diff
@@ -30,14 +30,6 @@ public static void main(String[] args) throws Exception {
 		Properties props = new Properties();
 		props.load(new FileInputStream(pathToConfig));
 		EventsInferrer.inferEvents(props, speciesCode);
-
-		// Link report file to report_ortho_inference.txt in website_files_update directory
-		/*
-		Path pathtoWebsiteFilesUpdateWithReportFile = Paths.get(props.getProperty("pathToWebsiteFilesUpdateFolder") + "report_ortho_inference.txt");
-		Path reportFilename = Paths.get("report_ortho_inference_test_reactome_" + props.getProperty("releaseNumber") + ".txt");
-		Files.deleteIfExists(pathtoWebsiteFilesUpdateWithReportFile);
-		Files.createLink(pathtoWebsiteFilesUpdateWithReportFile, reportFilename);
-		*/
```

Comment:
Just checking - this is removed because it is handled by Jenkins now?

Reply:
Thats correct. It's been sitting there commented out for a while now, and this was a good opportunity to remove it